pages/WatchList.js

```
@@ -117,76 +90,91 @@ const WatchList = ({ navigation }) => {
+    setAccountId(storedAccountId);
+    setSessionId(storedSessionId);
+  }, []);
 
   useEffect(() => {
-    {
-      sessionId ? getWatchListMovies(accountId, sessionId) : null;
+    if (sessionId) {
+      getWatchListMovies(accountId, sessionId);
     }
-  }, [sessionId]);
+  }, [sessionId, accountId, getWatchListMovies]);
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)